cogs/Feesh/feesh.py

```diff
@@ -423,17 +423,25 @@ def isvalid(m):
                 await channel.send(out_str)
                 raise ValueError('Target has no feesh')
 
-            def odds(x):
-                if x <= 0:
-                    return 1
-                else:
-                    # x > 0
-                    # 1 to 0.5
-                    return (np.exp(-x / 2) + 1) / 2
-
-            fail = odds(feesh_diff)
-
-            drop = (1 - fail) / 2  # success = 1. - fail - drop
+			def odds(y):
```

Comment:
You should be using a 4-spaces tab, otherwise the interpreter will complain

Reply:
Didn't realise, I think I was trying to not spam space by using tabs.